### ISSUE TITLE
Design | Profile | Accessibility | Content Updates for Modal when Editing Profile

### ISSUE THREAD

**Issue by SKasimow** [department-of-veterans-affairs/va.gov-team#46788]
#### Background:  revisit the text displayed in the "Are you sure?" modal (displayed when editing the profile and moving to a different field without either saving or canceling first).

Recommendations from Accessibility team:

> - Maybe instead of Continue and Cancel, the modal could be Are you Sure? with Yes/No buttons
> - On the other hand, as is might be better for those with cognitive disabilities

#### Tasks:
- [ ] Collaborate with content folks on improvements

**Modal 1:**
![Screen Shot 2022-09-09 at 11.26.22 AM.png](https://images.zenhubusercontent.com/5fb57aafa11eae0592a15cb1/53061308-026e-4154-9245-d5077ed9954e)

#### Acceptance Criteria:
- [ ] Review recommendations with a11y and Profile team and determine what (if any) changes should be made

**Comment by andaleliz** [comment]
Created [46894](https://github.com/department-of-veterans-affairs/va.gov-team/issues/46894) for content review.

**Comment by andaleliz** [comment]
@SKasimow Danielle approved the changes! [Here's a link to the mock-up](https://www.sketch.com/s/59857eb5-d9f9-4145-99d3-d9a1de2d0655/a/v8kxv7J)

The primary button label should be "Yes, cancel my changes", and close the modal, and bring the user back to read mode without their changes saved

The secondary button label should be "No, go back to editing" and close the modal and bring the user back to edit mode with their changes in progress.

**Comment by SKasimow** [comment]
@andaleliz  - thanks, opened #46935 (in draft)
